analysis/pseudobulk-bulk-prediction/README.md

```diff
@@ -6,7 +6,15 @@ To run the analysis, run:
 bash run-prediction.sh
 ```
 
-If you use 1Password to manage your credentials, you may need to run:
+If you are a Data Lab member using 1Password to manage your credentials, you may need to run:
```

Comment:
Other people could use 1Password too! (thought they won't have access no matter how they manage their credentials)

Reply:
> though they won't have access no matter how they manage their credentials

this was my logic right here!